5-flexbox/assets/index.html

```diff
@@ -1,99 +0,0 @@
-<!DOCTYPE html>
```

Comment:
you should not delete this file, you should have used this file. please rectify.